### PR TITLE
FIX: Case sensitive issue

### PR DESCRIPTION
**Pull request by johankladder**
In this commit the case sensitive issue for the "get playlist" is fixed.

### CODE REVIEW

File: src/nl/hanze/gameserver/server/command/GetCommandHandler.java [hunk 1]
```diff
@@ -17,10 +17,21 @@ public GetCommandHandler() {
...
+
+		String commandArgument = command.getArgument();
+		if(commandArgument != null) {
+			commandArgument = commandArgument.toLowerCase();
+		}
+
+		if(commandArgument.equalsIgnoreCase("gamelist")) {
 			client.writeResponse(Response.OK);
 			client.writeResponse(new Response(Response.Status.SVR, String.format("GAMELIST %s", getGameListString())));
-		} else if(command.getArgument().equals("playerlist")) {
```
Comment (IcyPalm) [inline src/nl/hanze/gameserver/server/command/GetCommandHandler.java]: When checking the gamelist command the code now uses: equalsIgnoreCase(), I suspect that this method also fixes the playerlist command. Try changing it to that

**Comment by Jarinus** [comment]
Looks good to me.

**Comment by goto-bus-stop** [comment]
Fix looks good! The indentation is off here though, using 4 spaces, whereas the rest of the code uses tabs. Please change that back to tabs, then it's ready to go :checkered_flag: 

(There might be an autoformat setting for indents in your IDE)

**Comment by goto-bus-stop** [comment]
:100: thanks!

**Pull request merged**